commit db083fa4bd681b6a2f35f96cd4de3df7527391f9 (refs/changes/81/3381/2)
Author: Josh Wilsdon <josh@wilsdon.ca>
Date:   2018-02-16T11:23:09-08:00 (1 year, 8 months ago)
    
    TRITON-128 add cnapi support for bhyve
    Reviewed by: Richard Kiene <richard.kiene@joyent.com>
    Approved by: Richard Kiene <richard.kiene@joyent.com>

diff --git a/lib/endpoints/vms.js b/lib/endpoints/vms.js
index 414e651..f56651f 100644
--- a/lib/endpoints/vms.js
+++ b/lib/endpoints/vms.js
@@ -548,7 +548,7 @@ VM.create = function handlerVmCreate(req, res, next) {
     var imgapi_url = req.stash.app.config.imgapi.url;
     var image_uuid;
 
-    if (req.params.brand === 'kvm') {
+    if (['bhyve', 'kvm'].indexOf(req.params.brand) !== -1) {
         if (req.params.hasOwnProperty('disks') && req.params.disks[0]) {
             image_uuid = req.params.disks[0].image_uuid;
         }
diff --git a/package.json b/package.json
index 5e33dca..adc3dba 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "cnapi",
   "description": "SmartDataCenter Compute Node API",
-  "version": "1.6.1",
+  "version": "1.7.1",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
